submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/repositories/Repositories.kt

```diff
@@ -75,3 +77,10 @@ interface AccessPermissionRepository : JpaRepository<AccessPermission, Long> {
     fun existsByAccessTagInAndAccessType(accessTags: List<String>, accessType: AccessType): Boolean
     fun findByUserIdAndAccessType(userId: Long, accessType: AccessType): List<AccessPermission>
 }
+
+interface UserDataDataRepository : JpaRepository<UserData, UserDataId> {
+    @Query("From UserData ud WHERE ud.userId = :userId and lower(ud.key) LIKE %:dataKey%")
```

Comment:
It's just a style thing but, since `WHERE` and `LIKE` are uppercase, probably `FROM` should be too